opengever/repository/browser/excel_export.py

```diff
@@ -141,8 +142,20 @@ def generate_report(request, context):
         u'label_groupnames_with_local_or_inherited_taskresponsible_role',
         default=u'Task responsible',
     ))
+
+    label_repositoryfolder_UID = repository_translater(_(
```

Comment:
naming should be lowercase with underscores.
```suggestion
    label_repositoryfolder_uid = repository_translater(_(
```